deploy/ansible/roles/wordpress-install/tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+# Include wordpress specific configuration
+- include_vars: "{{ item }}"
+  with_items:
+    - wordpress.yml
+    - version.yml
+  tags:
+    - configuration
+    - update
+
+- name: General | Install Application packages
+  apt: "name={{ item }} state=present"
+  with_items:
+    - git
+    - curl
+
+- name: WordPress | Ensure MySQL is running
+  command: service mysql start
+  ignore_errors: yes
+  tags:
+    - configuration
+
+- name: WordPress | Create WordPress database
+  mysql_db:
+    login_user: root
+    login_password: "{{ mysql_root_password }}"
+    db: "{{ wordpress_db_name }}"
+    state: present
+
+- name: WordPress | Create WordPress database user
+  mysql_user:
+    name: "{{ wordpress_db_user }}"
+    password: "{{ wordpress_db_password }}"
+    priv: "{{ wordpress_db_name }}.*:ALL,GRANT"
+    state: present
+    login_user: root
+    login_password: "{{ mysql_root_password }}"
+
+- name: WordPress | Download application from Github
+  sudo: false
+  git:
+    repo: "https://github.com/{{ application_repository }}/wordpress.git"
+    dest: "{{ wordpress_root_path }}"
+    version: "{{ application_version }}"
+    accept_hostkey: yes
+    ssh_opts: "-o StrictHostKeyChecking=no"
+    update: yes
+    force: yes
+  tags:
+    - update
+
+- name: WordPress | Current commit hash
+  shell: 'git log -n 1 --pretty=format:"%H" > {{ wordpress_root_path }}/.git-version'
+  args:
+    chdir: "{{ wordpress_root_path }}"
+  tags:
+    - update
+
+- name: Git | Ignore files permissions changes
+  command: "git config core.filemode false chdir={{ wordpress_root_path }}"
+  tags:
+    - update
+
+- name: Ansible | Update scripts
+  copy:
+    src: "{{ wordpress_root_path }}/deploy/ansible"
+    dest: /etc/
+  tags:
+    - update
+
+- name: WordPress | Configure Apache virtual host
+  template:
+    src: etc-apache24-confd-wordpress-conf
+    dest: /etc/apache2/sites-available/wordpress.conf
+    owner: www-data
+    group: www-data
+    mode: 0755
+
+- name: WordPress | Enable Apache virtual host
+  file: src=/etc/apache2/sites-available/wordpress.conf dest=/etc/apache2/sites-enabled/wordpress.conf state=link
+
+- name: Apache | Restart the Apache service
+  service:
+    name: apache2
+    state: restarted
+
+- name: WordPress | Configuration file config.php
+  template:
+    src: wp-config.php
+    dest: "{{ wordpress_root_path }}/wp-config.php"
+    owner: www-data
+    group: www-data
+    mode: 0500
+  # tags:
```

Comment:
I'd say the configuration tag is required here in case database credentials are changed